src/reference/informative/informC05_variable_reference.rst

```diff
@@ -8,4 +8,55 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding variable references
+
+    A :code:`variable` item only exists within its parent :code:`component` item, but can be connected to others via the :code:`map_variables` functionality.
+    There are two different ways that any :code:`variable` could be referenced.
+
+    The first and simplest is within its own local scope, the parent :code:`component`.
+    In that situation, the :code:`variable`\'s :code:`name` attribute is enough to uniquely locate it.
+
+    The second situation is when a :code:`variable` is referred to from a :code:`component` other than its parent, so a reference to the parent :code:`component` as well as the :code:`variable`'s :code:`name` is required.
+    This is found when creating :code:`connection` items using :code:`map_variables`.
+
+    The example below shows how the pairing of components and variables are required to form a valid connection.  
+
+
+    .. code-block:: xml
+
+        <model name="FamousFreds">
+
+          <component name="Flintstone">
+            <variable name="Fred" units="dimensionless" />
+            <variable name="Wilma" units="dimensionless" />
+          </component>
+
+          <component name="Astair">
+            <variable name="Fred" units="dimensionless" />
+          </component>
+
+          <component name="CartoonCharacters">
+            <variable name="FredFlintstone" units="dimensionless" />
+            <variable name="DaffyDuck" units="dimensionless" />
```

Comment:
There are many dimensions to Daffy Duck.

Reply:
So very true :)